templates/model/inputs.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"gopkg.in/yaml.v3"
+)
+
+// This file parses a YAML file that describes template input.
+
+// InputValue represents one of the parsed "input" fields from the inputs.yaml file.
```

Comment:
I know we agreed to naming this inputs.yaml in our meeting, but now I'm wondering if it should be something more general, like "test.yaml". If we ever decide to add fields to the YAML (like `test_description` or `test_flags`) then the name `inputs.yaml` would be misleading.

Reply:
Good call, done!